x/gov/abci.go

```diff
@@ -17,60 +17,70 @@ import (
 )
 
 // EndBlocker called every block, process inflation, update validator set.
-func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) error {
+func EndBlocker(ctx sdk.Context, k *keeper.Keeper) error {
```

Comment:
```suggestion
func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) error {
```

can you set this variable back to keeper so the diff is easier to view?